### PR TITLE
Ass about the project section

### PR DESCRIPTION
**Pull request by Tarek994**
- This PR includes the complete design for the about page.
- This design is made to showcase the details about our new project.
- I used TSX and CSS for its design.

### CODE REVIEW

File: src/components/pages/about/About.tsx [hunk 1]
```diff
@@ -0,0 +1,368 @@
+import "./about.css"
+import one from "./about-assets/one.svg"
+import two from "./about-assets/two.svg"
+import target from "./about-assets/target.svg"
+
+
+const about = () => {
```
Comment (alighali85) [inline src/components/pages/about/About.tsx]:
```suggestion
const About = () => {
```

**Comment by alighali85** [comment]
Good work, LGTM, I left a small comment on the code, also this file `src/components/pages/home/home.css` looks like it's empty you may remove it if it's not needed.

File: src/components/pages/about/About.tsx (hunk 1 above)
Comment (Tarek994) [reply src/components/pages/about/About.tsx]: This problem has been resolved.

**Comment by Tarek994** [comment]
> Good work, LGTM, I left a small comment on the code, also this file `src/components/pages/home/home.css` looks like it's empty you may remove it if it's not needed.

**Pull request merged**